job_templates/snippet/property_parameter-definition_common.xml.em

```diff
@@ -44,7 +44,7 @@ To use the latest released version, use an empty string.</description>
             <a class="string-array">
               <string>@ubuntu_distro</string>
 @{
-choices = ['bionic', 'xenial']
+choices = ['bionic', 'xenial', 'buster']
```

Comment:
I added this line for manual CI.
Should I delete it?

Reply:
I think we want to be able to run buster CI if it's available. :+1: for keeping it.